contracts/RiskManagerV2.sol

```diff
@@ -0,0 +1,158 @@
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import "./interfaces/IRiskManagerV2.sol";
+import "./Auctioneer.sol";
+import "./GovernanceUtils.sol";
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/// @title Risk Manager for tBTC v2
+/// @notice Risk Manager is a smart contract with the exclusive right to claim
+///         coverage from the coverage pool. Demanding coverage is akin to
+///         filing a claim in traditional insurance and processing your own
+///         claim. The risk manager holds an incredibly privileged position,
+///         because the ability to claim coverage of an arbitrarily large
+///         position could bankrupt the coverage pool.
+///         tBTC v2 risk manager demands coverage by opening an auction for TBTC
+///         and liquidating portion of the coverage pool when tBTC v2 deposit is
+///         in liquidation and signer bonds on offer reached the specific
+///         threshold. In practice, it means no one is willing to purchase
+///         signer bonds for that deposit on tBTC side.
```

Comment:
I think we had a communication glitch. For v2, we do not want to run auctions. It's as simple as having some multisig being able to claim the coverage. From [TIP-043](https://forum.threshold.network/t/tip-043-coverage-pool-migration/465) proposal:

> - Treasury Guild becomes risk manager on the T-only Coverage Pool. 
> - This is the group responsible for claiming coverage out of the pool if needed. 
> - The Treasury Guild has autonomy over how this role operators within the DAO and can make changes when needed.
> - The DAOs funds in T are at risk only so the can call coverage from pool without limit
> - Move fast in event where coverage needs to be claimed. Not time for full DAO ratification
> - Further discussion on an extended risk manager role is discussed in [this forum thread](https://forum.threshold.network/t/risk-manager-for-tbtcv2/463). This content of this discussion was determined to be outside of the scope of this proposal

That being said, I do not think we need `RiskManagerV2` at all. If the Treasury Guild multisig is an approved risk manager in `CoveragePool`, they can call `seizeFunds` (or `seizePortionOfPool` / `seizeAmountFromPool` directly.